rabbitmq/conf.yaml.example

```diff
@@ -20,12 +20,12 @@ instances:
     #
     # ssl_verify: false
 
-    # The (optional) no_proxy parameter would bypass any proxy settings enabled
+    # The (optional) skip_proxy parameter would bypass any proxy settings enabled
```

Comment:
I think we're missing the changes to the actual rabbitmq check, like you did here: https://github.com/DataDog/integrations-core/pull/1057/files#diff-4d1da4a0950d6778c2c2b221ea16ab1dR193

Also, we'd have to bump the bugfix semver version of either check.

Reply:
Sorry, what's missing?

I'll change the version now.

Reply:
This has actually been removed from rabbit

Reply:
Huh, yes it's no longer there in the actual check but nothing about it in the changelog. Why was it removed? I'll look through the history.

Reply:
No it's still there https://github.com/DataDog/integrations-core/blob/master/rabbitmq/datadog_checks/rabbitmq/rabbitmq.py#L337

See the prerequisite PRs

Reply:
Oop nevermind!